src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/enums/SessionConflict.kt

```diff
@@ -3,5 +3,5 @@ package uk.gov.justice.digital.hmpps.visitscheduler.dto.enums
 @Suppress("unused")
 enum class SessionConflict {
   NON_ASSOCIATION,
-  DOUBLE_BOOKED,
+  DOUBLE_RESERVATION,
```

Comment:
Question: will this require a frontend change, do they implement this enum?

Reply:
will need updating on the Orchestration Service.

Reply:
DOUBLE_BOOKING_OR_RESERVATION